frontends/mit-open/src/page-components/ChannelDetails/ChannelDetails.tsx

```diff
@@ -89,8 +89,8 @@ const getFacetManifest = (channelType: ChannelTypeEnum) => {
       name: "more_information",
       title: "More Information",
       labelFunction: (key: string, channelTitle: string) => (
-        <a href={key}>
-          {channelTitle} website <RiExternalLinkLine size={18} />
+        <a target="_blank" href={key} rel="noopener noreferrer">
```

Comment:
```suggestion
        <a target="_blank" href={key} rel="noopener">
```

Reply:
`noreferrer` implies `noopener`...  I meant to suggest just `noopener` What do you think?

Reply:
ah wierd. i thought i had committed just noopener. merging your edit.